src/adaptor/mod.rs

```diff
@@ -0,0 +1,21 @@
+mod ini;
+
+use std::collections::HashMap;
+use std::io::{Read, Write};
+
+#[derive(Debug, Eq, PartialEq)]
+pub enum Config {
```

Comment:
Yes, it's probably not the best name. Any suggestions for a different one? Maybe just `Value`?